tests/test.nf.test

```diff
@@ -24,7 +24,7 @@ nextflow_pipeline {
                         path("$outputDir/centrifuge/"),
                         path("$outputDir/diamond/database.dmnd"),
                         path("$outputDir/ganon/database.hibf"),
-                        path("$outputDir/ganon/database.tax").readLines().last().contains('Haemophilus_influenzae'),
+                        path("$outputDir/ganon/database.tax").readLines().last().contains('SARS-CoV-2_genome'),
```

Comment:
Why does this PR change the test for ganon?

Reply:
The new test data adds 4 more genomes (I have to combine the two changes: krakenuniq and test data update due to the test-data already being merged in)